Make.inc

```diff
@@ -1002,9 +1002,16 @@ MKLLIB := $(MKLROOT)/lib/ia32
 endif
 USE_SYSTEM_BLAS:=1
 USE_SYSTEM_LAPACK:=1
+ifeq ($(OS), WINNT)
+LIBBLASNAME := mkl_rt
+LIBLAPACKNAME := mkl_rt
+# work around libtool issue with arpack
+MKL_LDFLAGS := -L$(MKLLIB) -Wl,-lmkl_rt
```

Comment:
This was part of the reason I tried in https://github.com/tkelman/julia/commit/812c1840f4e7970e3a015b6be87e3443ecd9cdcc to not use autotools at all for building arpack, but that had some issues on mac (see discussion in https://github.com/JuliaLang/julia/pull/14518). Arpack needs to be moved to a package so this isn't worth too much worrying about as long as there's an easy workaround for now.